ambry-utils/src/main/java/com/github/ambry/utils/Time.java

```diff
@@ -24,12 +25,10 @@
   /**
    * Some common constants
    */
-  public static final int NsPerUs = 1000;
-  public static final int UsPerMs = 1000;
-  public static final int MsPerSec = 1000;
-  public static final int NsPerMs = NsPerUs * UsPerMs;
-  public static final int NsPerSec = NsPerMs * MsPerSec;
-  public static final int SecsPerMin = 60;
+  public static final int MsPerSec = Math.toIntExact(TimeUnit.SECONDS.toMicros(1));
```

Comment:
Hi @djKooks, for this issue, my thought was that these constants could be removed from our codebase entirely and replaced with usages of `TimeUnit` inline. Doing so will help to avoid integer overflow issues and typos from having to chain a lot of unit conversions.

Would you be open to exploring the codebase and finding places where these constants are used?

Reply:
@cgtz oh, I see. 
1. So you mean to remove variables such as `MsPerSec`, `SecsPerMin` in entire project?
2. Will it be okay to convert all values to `Long` type?
3

Reply:
1. Yep, that's what I mean.
2. I think in most cases, time should be a `long`. If there are edge cases where time is represented as an `int`, feel free to ask about it on this thread and I can comment about what would be best in those situations.

On another note, thank you for contributing to Ambry! Do you personally deploy ambry? We are trying to engage more with our open source community.

Reply:
@cgtz thanks for comment. I'll try to update it.

Not personally deploying it, but maybe could in future. I'm pretty interested in it. If there's any future plan or update, please let me know 🙏